### PR TITLE
Use specs and secrets

### PR DESCRIPTION
**Pull request by pierluigi**
This PR refactors our samples to 
- [use secrets](https://docs.fission.io/0.9.1/usage/access-secret-cfgmap-in-function) instead of passing pem files and app IDs around
- [use specs](https://docs.fission.io/0.9.1/tutorial/developer-workflow) rather than abusing the fission CLI for declarative deploys
- rewrite the README for simplicity
- move the README to the samples folder (makes more sense anyway)

I didn't create a spec for the issue-to-pr function as we'll need to revisit that example anyway.

@leefaus 

👀 please review and 👍 validate my instructions. 
⚠️  Please do a `fission spec destroy` before replaying the steps in the README to start anew.

### CODE REVIEW

File: samples/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* `eval $(minikube docker-env) && docker build --no-cache -t github/hubotjs .`
+
+### Run our issue comment example
+
+* `cd ../samples`
+* `fission spec apply --wait`
+
+* setup NGROK to point to the fission ROUTER URL
+  * `minikube service list`
+  * `ngrok http [LOCAL_ROUTER_IP:PORT]`
```
Comment (leefaus) [inline samples/README.md]: This is awesome work @pierluigi.  I wonder if we could use this as part of the setup:

`export FISSION_ROUTER="$(minikube service --url -n fission router)"`

Then we would do `ngrok http $FISSION_ROUTER` here. 

What do you think?

**Review by leefaus — CHANGES_REQUESTED** [review]
Can you test my hypothesis in this reveiw?

File: samples/README.md (hunk 1 above)
Comment (pierluigi) [reply samples/README.md]: Great idea – `ngrok http ${$(minikube service --url -n fission router)#*//}` is what we want though! Pushing a new commit.

**Review by leefaus — APPROVED** [review]
Look at you getting all fancy with your `bash` skillz.  💯

**Pull request merged**